lms/djangoapps/courseware/views/views.py

```diff
@@ -848,6 +852,8 @@ def course_about(request, course_id):
 
         sidebar_html_enabled = course_experience_waffle().is_enabled(ENABLE_COURSE_ABOUT_SIDEBAR_HTML)
 
+        allow_anonymous = COURSE_ENABLE_UNENROLLED_ACCESS_FLAG.is_enabled(course_key)
```

Comment:
We also need to check the course_visibility setting.

Reply:
Done